app/views/trips/index.html.erb

```diff
@@ -2,46 +2,101 @@
 
 <h1 class="margin-title-lg">My Trips</h1>
 
-<div class="cards">
-  <% @trips.each do |trip| %>
-    <%= link_to trail_path(trip.trail), class:"card my-card-link" do %>
-      <div class= "image-container">
-        <%= cl_image_tag trip.trail.photos.first.key, class: "rounded-3 card-image", alt: "trail picture" %>
-        <%# <img src="https://lonelyplanetimages.imgix.net/a/g/hi/t/a2aa48e66952bf8816898991072e32f5-macritchie-reservoir.jpg" alt="trail picture" class="rounded-3 card-image"> %>
-        <i class="fa-regular fa-bookmark"></i>
-      </div>
-      <div class ="card-body title-body d-flex align-items-start">
-        <div class="title-rating pt-2 mt-0">
-          <h5 class="card-title">
-            <%= trip.trail.name %>
-          </h5>
-        </div>
-        <div class="d-flex pt-2 align-items-center">
-          <p class="card-text mb-0">3.7</p>
-          <i class="fa-solid fa-star ms-1"></i>
-        </div>
-      </div>
+<div data-controller="switch-tabs">
+  <div class="cust-container">
+    <div class="d-flex mb-3">
+      <button class="trip-category-select active me-2" data-action="click->switch-tabs#upcoming" data-switch-tabs-target="upcomingbutton"><h4 class="mb-1">Upcoming</h4></button>
+      <button class="trip-category-select" data-action="click->switch-tabs#past" data-switch-tabs-target="pastbutton"><h4 class="mb-1">Past</h4></button>
+    </div>
+  </div>
 
-      <%# <div style="height: 200px; width: 300px; background-position: center; background-repeat: no-repeat; background-size: cover; background-image: url('https://lonelyplanetimages.imgix.net/a/g/hi/t/a2aa48e66952bf8816898991072e32f5-macritchie-reservoir.jpg')">
```

Comment:
Would it be possible to use the cards partial? (Use partial but just add trip date on top)
--> I think the easiest way would be to just add the date (if that's the only difference) inside the partial, with a condition that the trip date only appears if it has a date in the future, or something like that (but then in the index, you would still see the date for upcoming trips)
So that the changes we use for the cards are reflected here, since I have changed the cards.